agent/php_nrini.c

```diff
@@ -222,6 +223,53 @@
 
 ZEND_DECLARE_MODULE_GLOBALS(newrelic)
 
+#define NR_INI_PREFIX_LEN 9
+
+char* nr_ini_to_env(const char* ini_name) {
+  char* env_name = NULL;
+  char* buf = NULL;
+  char* ini_upper = NULL;
+  const char* NR_PREFIX_STR = "NEW_RELIC";
```

Comment:
I am not sure about using all caps for a local variable name.  I see what you are going for here.  If we want to go with all caps because this is a constant then can we move `NR_INI_PREFIX_LEN` into as a local as well defined as 

```
size_t NR_INI_PREFIX_LEN = nr_strlen(NR_PREFIX_STR);
```
I would like a solution where it isn't necessary to update the length if the string is ever changed.

Reply:
> I am not sure about using all caps for a local variable name.

I elected to do it this way to cut down on what I'd call `#define` pollution- saves the effort of having to undef something that only has to be used once or in a single function. As for all caps, I like the idea of the nature of the variable (being a constant) being abundantly clear vs a non-const variable.

> If we want to go with all caps because this is a constant then can we move NR_INI_PREFIX_LEN into as a local as well

We can (and maybe should) do this- but the provided solution is not quite correct- `NR_INI_PREFIX_LEN` is meant to capture the length of the string `newrelic.`, not `NEW_RELIC`. It just happens that they are currently the same length, but as you point out, that may not be the case.

Reply:
Got it - I'd like the (corrected) change if you're good with it.

Reply:
Cleaned up constants here: 611b2f0b54e66b70c3074b87182c058006ee88d2